### PR TITLE
Allow configuring when Celery tasks should expire

### PR DESCRIPTION
**Pull request by csomh**
Allow an expiry time to be configured for the Celery tasks created by
the scheduled update job.

This will be useful to help avoiding a forever growing Celery task
queue.

Use a value of 23 hours as a default value in the deployment role.

Signed-off-by: Hunor Csomortáni <csomh@redhat.com>

### CODE REVIEW

File: dist2src/worker/updater.py [hunk 1]
```diff
@@ -125,6 +125,12 @@ def _create_task(self, project: PagureProject, branch: str, commit: str):
...
         logger.debug(f"Sending task {task_name!r}, with payload: {event}")
-        r = celery_app.send_task(name=task_name, kwargs={"event": event})
-        logger.info(f"Task UUID={r.id} sent to Celery.")
+        r = celery_app.send_task(
+            name=task_name,
+            expires=self.cfg.update_task_expires,
+            kwargs={"event": event},
+        )
+        logger.info(
+            f"Task UUID={r.id}, expiring in {self.cfg.update_task_expires} seconds sent to Celery."
```
Comment (jpopelka) [inline dist2src/worker/updater.py:134]: Having "expiring in 82800 seconds" in each of these lines doesn't bring much value (to me).
What about:
```suggestion
            f"Task UUID={r.id}, expiring in at {datetime.now() + timedelta(seconds=self.cfg.update_task_expires)} sent to Celery."
```

**Review by jpopelka — APPROVED** [review]

**Review by TomasTomecek — APPROVED** [review]
LGTM

File: dist2src/worker/updater.py (hunk 1 above)
Comment (csomh) [reply dist2src/worker/updater.py]: Good point, thanks!

Made an update to log this value when `check_updates` starts, next to the other configuration values. WDYT?

**Comment by csomh** [comment]
> Build failed.

> * [dist2src-tests ](https://softwarefactory-project.io/zuul/t/packit-service/build/86a0d37a89b14133b7386c5b801cd2da) : FAILURE in 26m 37s

The failure of `tests/test_update.py::test_update[alsa-lib-c8s]` seems to have been a flake. It passed locally.

**Review by jpopelka — APPROVED** [review]

**Pull request merged**